FEATURES.md

```diff
@@ -0,0 +1,175 @@
+# Features
+
+This document describes a list of use-cases and features that this library provides.
+
+## Use Cases
+
+The following are the core use cases that this integration supports. If you have another use case, please let us know by creating an Issue.
+
+* UC 1: As a user, I want to trigger a Keptn quality gate evaluation from within Jenkins
```

Comment:
I know it is picky, but please change the wording to: 

* UC 1: As a user, I want to perform a release/deployment validation by triggering a Keptn quality gate evaluation from within Jenkins. 

---

The release/deployment validation is what I would like to do. The quality gate is just the technical vehicle for achieving that.

Reply:
I didn't use "release/deployment validation" on purpose, because a Jenkins/Keptn user might not be aware of what a "release/deployment validation" is.

Though I'm happy to use your suggestion if we can provide an external link explaining release/deployment validation.

Reply:
Not available yet - but we mention "SLO/SLI validation" here: https://keptn.sh/docs/concepts/performance_validation/#test-automation-process with a reference to https://keptn.sh/docs/concepts/quality_gates/

With minor adoptions of the `Quality Gate` page, we can strengthen the message of doing release/deployment validation. I will provide a PR for updating the keptn.sh docs there.

Reply:
I filled a PR on the keptn.sh page. May I ask you for a review: https://github.com/keptn/keptn.github.io/pull/978/

Reply:
Done